optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/GizmoMemberAccessorEntityEnhancer.java

```diff
@@ -178,41 +179,33 @@ public static String generateFieldAccessor(AnnotationInstance annotationInstance
                 Thread.currentThread().getContextClassLoader());
         Field fieldMember = declaringClass.getDeclaredField(fieldInfo.name());
         String generatedClassName = GizmoMemberAccessorFactory.getGeneratedClassName(fieldMember);
-        try (ClassCreator classCreator = ClassCreator
-                .builder()
-                .className(generatedClassName)
-                .interfaces(MemberAccessor.class)
-                .classOutput(classOutput)
-                .setFinal(true)
```

Comment:
Is the generated class still final?
Rumor is the JVM hotspot inliner likes to know if classes are final - although it inspects the classpath to know if classes are "effectively final".

Reply:
It is. I did not remove or change this code, it just moved to another place - and at the same time, I reduced some code duplication.

Reply:
Nevermind I see it still is :)